gui-libs/vte-common/vte-common-0.76.3.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{10..13} )
+
+inherit gnome.org meson python-any-r1
+
+DESCRIPTION="Library providing a virtual terminal emulator widget"
+HOMEPAGE="https://wiki.gnome.org/Apps/Terminal/VTE"
```

Comment:
https://gitlab.gnome.org/GNOME/vte

Reply:
Done in https://github.com/gentoo/gentoo/pull/34844/commits/01d9d48117482792340739f4f7452598cb90f7f9